From 7f7a9cec379bff8e06daa56496b86790bfca59e7 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Tue, 27 Feb 2018 16:43:30 -0800
Subject: [PATCH] TRITON-177 some Triton components are playing with fire and
 should update to sdcnode build from TOOLS-1990 Reviewed by: Trent Mick
 <trentm@gmail.com> Approved by: Trent Mick <trentm@gmail.com>

---
 Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index f8cc43e..f695090 100644
--- a/Makefile
+++ b/Makefile
@@ -32,11 +32,10 @@ JSL_FILES_NODE =	$(JS_FILES)
 JSSTYLE_FILES =		$(JS_FILES)
 JSSTYLE_FLAGS =		-o indent=4,doxygen,unparenthesized-return=0
 
-NODE_PREBUILT_TAG =	gz
 NODE_PREBUILT_VERSION =	v0.10.48
 
 ifeq ($(shell uname -s),SunOS)
-NODE_PREBUILT_TAG =	zone
+NODE_PREBUILT_TAG =	gz
 # sdc-minimal-multiarch-lts 15.4.1
 NODE_PREBUILT_IMAGE =	18b094b0-eb01-11e5-80c1-175dac7ddf02
 endif
-- 
2.21.0

